Cargo.toml

```diff
@@ -64,7 +64,8 @@ trust-dns-resolver = { version = "0.23", optional = true }
 portpicker = { version = "0.1", optional = true }
 num-bigint-dig = "0.8"
 tracing = "0.1"
-rustls = { version = "0.21", features = ["dangerous_configuration"], optional = true }
+rustls = { path = "../deleteIt/rustls/rustls", optional = true }
+#rustls = { version = "0.22.2", optional = true }
```

Comment:
issue: This intermediate commit is referring to a local fork. This commit should be amended to pass the CI if we want to rebase.